src/content/reference/react/createFactory.md

```diff
@@ -22,57 +22,57 @@ const factory = createFactory(type)
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `createFactory(type)` {/*createfactory*/}
 
-Call `createFactory(type)` to create a factory function which produces React elements of a given `type`.
+Panggil `createFactory(type)` untuk membuat fungsi pabrik yang menghasilkan elemen-elemen React dengan `type` yang ditentukan.
 
 ```js
 import { createFactory } from 'react';
 
 const button = createFactory('button');
 ```
 
-Then you can use it to create React elements without JSX:
+Lalu Anda dapat menggunakannya untuk membuat elemen-elemen React tanpa JSX :
 
 ```js
 export default function App() {
   return button({
     onClick: () => {
-      alert('Clicked!')
+      alert('Sudah diklik!')
     }
-  }, 'Click me');
+  }, 'Klik saya');
 }
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh-contoh lainnya di bawah ini.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `type`: The `type` argument must be a valid React component type. For example, it could be a tag name string (such as `'div'` or `'span'`), or a React component (a function, a class, or a special component like [`Fragment`](/reference/react/Fragment)).
+* `type`: Argumen `type` harus merupakan tipe komponen React yang valid. Misalnya, bisa berupa string nama tag (seperti 'div' atau 'span'), atau komponen React (fungsi, kelas, atau komponen khusus seperti [`Fragment`](/reference/react/Fragment)).
```

Comment:
Do not remove the backtick

```suggestion
* `type`: Argumen `type` harus merupakan tipe komponen React yang valid. Misalnya, bisa berupa string nama tag (seperti `'div'` atau `'span'`), atau komponen React (fungsi, kelas, atau komponen khusus seperti [`Fragment`](/reference/react/Fragment)).
```

Reply:
Thank you for the review.

I'll fix it at the earliest opportunity.